Python/Product/Django/Intellisense/TemplateCommandFactory.cs

```diff
@@ -14,17 +14,14 @@
 // See the Apache Version 2.0 License for specific language governing
 // permissions and limitations under the License.
 
+#if !DEV16_OR_LATER
```

Comment:
If you're not worrying about Dev15 builds, can you just get rid of this file?

Reply:
Agreed, I'll remove the django dev15 only code.